app/src/Policy/GatheringPolicy.php

```
@@ -38,4 +38,9 @@ public function canViewAttendance(KmpIdentityInterface $user, BaseEntity $entity
 
         return $this->_hasPolicy($user, $method, $entity);
     }
+
+    public function canQuickView(KmpIdentityInterface $user, BaseEntity $entity, ...$optionalArgs): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)